ruby/vampires.rb

```diff
@@ -110,3 +110,5 @@
 end
```

Comment:
For Line 98:

Lets break the evaluation conditions out of the questions. It is always good to finish one job before starting another. Collect all of the information you need about your employee and then move to eval separately.